File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/DomainProvProcessPending.scala
package io.swagger.server.model


/**
 * Provisioning status data for a pending domain order.
 *
 * @param _OPS_version 
 * @param response_text 
 * @param protocol 
 * @param response_code 
 * @param action 
 * @param `object` 
 * @param is_success 
 * @param attributes 
 */
case class DomainProvProcessPending (
  _OPS_version: Option[String],
  response_text: Option[String],
  protocol: Option[String],
  response_code: Option[String],
  action: Option[String],
  `object`: Option[String],
  is_success: Option[String],
  attributes: Option[DomainProvProcessPending_attributes]
)

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/DomainProvProcessPending.js
goog.provide('API.Client.DomainProvProcessPending');

/**
 * Provisioning status data for a pending domain order.
 * @record
 */
API.Client.DomainProvProcessPending = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.DomainProvProcessPending.prototype.oPSVersion;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainProvProcessPending.prototype.responseText;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainProvProcessPending.prototype.protocol;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainProvProcessPending.prototype.responseCode;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainProvProcessPending.prototype.action;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainProvProcessPending.prototype.object;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainProvProcessPending.prototype.isSuccess;

/**
 * @type {!API.Client.DomainProvProcessPending_attributes}
 * @export
 */
API.Client.DomainProvProcessPending.prototype.attributes;

